lib/manager/types.ts

```diff
@@ -115,17 +94,14 @@ export interface Package<T> extends ManagerData<T> {
   repo?: string;
   target?: string;
   versioning?: string;
-
-  // npm manager
   bumpVersion?: ReleaseType | string;
-  npmPackageAlias?: boolean;
+  isPackageAlias?: boolean;
```

Comment:
Shouldn't this be moved to `managerData` too?

Reply:
Actually it's stripped out before the `PackageFile` is returned, so it doesn't belong in `PackageFile` or `managerData`